neo/Prompt/Commands/tests/test_config_commands.py

```diff
@@ -25,6 +36,9 @@ def test_config(self):
         self.assertFalse(res)
 
     def test_config_output(self):
+        # importing because it setups `peewee` logging, which is checked at the test below

```

Comment:
Should be "sets up" not "setups"

Reply:
Hey no worries! I only wish I could be of more help. It seems like you made a big push the other day!